qemu/tests/virtio_port_hotplug.py

```diff
@@ -35,6 +35,7 @@ def run(test, params, env):
     timeout = int(params.get("login_timeout", 360))
     module = params.get("modprobe_module")
     check_module = params.get_boolean("check_module", "yes")
+    bg_test = params.get_boolean("bg_test", "yes")
```

Comment:
typo: yes
```python
get_boolean(self, key, default=False)
```

Reply:
@PaulYuuu The function `get_boolean` accept `yes`,`on`,`true`

Reply:
The second parameter requires a bool type.
`value = self.get(key, "yes" if default else "no")`
```python
    def get_boolean(self, key, default=False):
        """
        Check if a config option is set to a default affirmation or not.

        :param key: config option key
        :type key: str
        :param bool default: whether to assume "yes" or "no" if param
                             does not exist (defaults to False/"no").
        :return whether option is set to 'yes'
        :rtype: bool
        """
        value = self.get(key, "yes" if default else "no")
        if value in ("yes", "on", "true"):
            return True
        if value in ("no", "off", "false"):
            return False
        raise ValueError("Cannot get boolean parameter value for %s: %s", key, value)
```

Reply:
@PaulYuuu ah, yes, my careless , thanks for point it out.

Reply:
thanks and update.